### PR TITLE
Mb/ifu missing

### PR DESCRIPTION
**Pull request by sesquideus**
I added more recipes whose implementation was missing (`metis_ifu_*`).
Also many more tests and we're now skipping those that refer to missing files (again in IFU).

### CODE REVIEW

**Comment by hugobuddel** [comment]
Perhaps ask one of the other recipe writers to review? Looks good to me though

**Comment by janusbrink** [comment]
Tested my assigned metis_ifu_rsrf recipe with some dummy input files and it runs fine - so it looks good to merge from my side...

File: metisp/pymetis/src/pymetis/base/product.py
Comment (janusbrink) [inline metisp/pymetis/src/pymetis/base/product.py:1]: L96: Why a warning message and not just 'info'?

**Review by janusbrink — APPROVED** [review]

**Comment by hugobuddel** [comment]
Great @sesquideus !

We should make sure we add any new (/newly working) recipes to
- https://github.com/AstarVienna/METIS_Pipeline/blob/main/metisp/pyrecipes/metis_recipes.py and
- https://github.com/AstarVienna/METIS_Pipeline/blob/main/.github/workflows/run_edps.yaml

Perhaps we're at the stage where we can remove the `-t metis_lm_img_basic_reduce` from the edps call in `run_edps.yaml`?

I placed that target there because the other recipes were not yet complete.

**Pull request merged**